src/PublicApiAnalyzers/Core/Analyzers/DeclarePublicApiAnalyzer.cs

```diff
@@ -195,13 +206,11 @@ public override void Initialize(AnalysisContext context)
         private void OnCompilationStart(CompilationStartAnalysisContext compilationContext)
         {
             var additionalFiles = compilationContext.Options.AdditionalFiles;
+            var errors = new List<Diagnostic>();
 
-            if (!TryGetApiData(additionalFiles, compilationContext.CancellationToken, out ApiData shippedData, out ApiData unshippedData))
-            {
-                return;
-            }
-
-            if (!ValidateApiFiles(shippedData, unshippedData, out List<Diagnostic> errors))
+            // TODO: Switch to "RegisterAdditionalFileAction" available in Microsoft.CodeAnalysis "3.8.x" to report additional file diagnostics.
```

Comment:
❔ File an issue for this?

Reply:
https://github.com/dotnet/roslyn-analyzers/issues/3918